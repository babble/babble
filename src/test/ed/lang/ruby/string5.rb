TOKEN_LENGTH = 4

characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890'
puts characters.length
puts characters[5]
puts characters[5..5]
puts characters[5..6]

# def random_token
#   characters = 'ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz1234567890'
#   temp_token = ''
#   TOKEN_LENGTH.times do
#     pos = 5#rand(characters.length)
#     puts pos
#     puts characters[5..5]
#     temp_token += characters[pos]
#   end
#   temp_token
# end

# puts random_token
