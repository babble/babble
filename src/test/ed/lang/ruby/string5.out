62
70
F
FG
